mountpoint-s3/src/main.rs

```diff
@@ -251,14 +251,6 @@ impl CliArgs {
 fn main() -> anyhow::Result<()> {
     let args = CliArgs::parse();
 
-    // validate mount point
```

Comment:
Are we okay moving this validation to the background process?

Reply:
We should be -- it was the only validation we were doing in the foreground process previously, and we shouldn't be doing any work at all before the fork.

Reply:
Makes sense, but we need to update the tests.